### PR TITLE
Rework UX of kit variant creation

### PR DESCRIPTION
**Pull request by mttdffy**
BUNDLES-21

### CODE REVIEW

File: app/view_models/workarea/admin/product_view_model.decorator [hunk 1]
```diff
@@ -14,6 +14,16 @@ module Workarea
         end
     end
 
+    def variant_details
```
Comment (bencrouse) [inline app/view_models/workarea/admin/product_view_model.decorator:17]: Seems like this is the same as this method, can you reuse it? https://github.com/workarea-commerce/workarea/blob/master/admin/app/view_models/workarea/admin/product_view_model.rb#L96

**Review by bencrouse — CHANGES_REQUESTED** [review]
There's a lot going on in that `Params` class, feels like some unit tests would be good to have on that, can you add some?

File: app/view_models/workarea/admin/product_view_model.decorator (hunk 1 above)
Comment (mttdffy) [reply app/view_models/workarea/admin/product_view_model.decorator:17]: that method is really problematic, and looks like we don't use it. Probably should get rid of it. Or at least rename it like this one.

**Comment by bencrouse** [comment]
Looks good, I'm going to wait to merge until I release v3.5.20.

File: workarea-product_bundles.gemspec [hunk 2]
```diff
@@ -20,5 +20,5 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.3.0'
 
-  s.add_dependency 'workarea', '~> 3.x', '>= 3.5.16'
+  s.add_dependency 'workarea', '~> 3.x', '>= 3.5.19' # TODO: should be 3.5.20
```
Comment (bencrouse) [inline workarea-product_bundles.gemspec:23]:
```suggestion
  s.add_dependency 'workarea', '~> 3.x', '>= 3.5.20'
```

**Review by bencrouse — APPROVED** [review]

**Pull request merged**